membership-baseline.md

```diff
@@ -0,0 +1,169 @@
+
+## Regular Membership:
+
+- Annual fee: $X
+- Full access to all community resources and benefits.
+
+## Student Membership:
+
+- Annual fee: $X (with proof of enrollment)
+- Discounted rate for students with a passion for security.
+
+## Lifetime Membership:
+
+- One-time payment: $X
+- Enjoy lifelong access to all community resources and benefits.
+
+# Membership Process:
+To become a member of null, please follow these steps:
+
+## Step 1: Submitting Membership Application:
+
+- WIP
+    - Ex: Where one can fill the form to become member
+    - What happens next ?
+
+## Step 2: Payment Processing:
+
+- WIP
+    - Where can they make the payment
+    - What types of payment modes are supported by the platform if used any
+
+## Step 3: Membership Record Management:
+
+- WIP
+    - How are we planning to keep record of the payment
+    - How do we plan to keep record of each member
+    - Is the provided solution scalable ?
+
+
+# Rules & Regulations:
+- Membership dues are the responsibility of the member and cannot be subsidized or funded by Null Community projects, chapters, or events.
+- Members are not affiliated with any specific chapter or project but are considered at-large members upon submission of the membership form.
+- Applicants give their consent to receive communications from the Null Community regarding their membership status.
+- Only eligible individuals will qualify for complimentary, regional, and student memberships.
+- The Null Community is the sole authority in granting memberships. No person or entity other than the Null Community's board can act as a membership agent or confer honorary or complimentary memberships.
+- Fraudulent membership submissions will be revoked by the Null Community without notice and without a refund. Memberships and associated benefits are non-transferable.
+
+
+To ensure a safe and inclusive community environment, we have established the following rules and code of conduct:
+
+## Respectful Communication:
+ 
+- Treat fellow members with respect, courtesy, and professionalism.
+- Engage in constructive discussions and refrain from personal attacks or offensive language.
+
+## Confidentiality and Responsibility:
+
+- Handle sensitive information shared within the community with utmost care and confidentiality.
+- Respect copyrights and intellectual property rights when sharing resources.
+
+## Compliance with Laws and Regulations:
+
+- Adhere to all applicable laws and regulations related to cybersecurity and ethical hacking.
+- Refrain from engaging in any illegal activities or discussions that promote unethical behavior.
+
+
+# Moderator Approval:
+To ensure the quality and integrity of our community, all membership applications are subject to review and approval by our moderators. The following process is followed:
+
+## Criteria for Approval:
+If yes, we come up with an approach here [WIP]
+
+## Approval Timeline:
+If yes, what will be the approval timeline?
```

Comment:
since we are marking board as the authority for acceptance we will have to setup a approval process automatic otherwise approval's can only happen when board is present.

Reply:
yet to decide